bp_be/test/tb/bp_be_dcache/py/bsg_ascii_to_rom.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python

```

Comment:
Use the one in basejump_stl

Reply:
We copied the file from basejump_stl, changed it so that it writes to a file (it was printing to the screen).

Reply:
In the Makefile use > .txt